src/App.tsx

```diff
@@ -60,8 +60,8 @@ class App extends Component {
   };
 
   configValidationCompletedHandler = async (config: Config) => {
-    this.setState({ isLoading: false });
     await this.initializeServices(config);
+    this.setState({ isLoading: false });
```

Comment:
This is causing flaky continue watching tests. Sometimes the watchlist is still loading while navigating to a video page.

Reply:
Thank you for investigating!